frontend/src/styles/partnerbar.scss

```diff
@@ -0,0 +1,102 @@
+@import "variables.scss";
+
+.page {
+    margin-top: 5%;
```

Comment:
Using margins on tags that surround a large potion of the page or all of the page is probably contributing to you having that extra whitespace at the bottom of your pages. Try using padding instead of margin for these cases instead.

Reply:
ok

Reply:
done